ms-common/src/main/java/net/trajano/ms/common/jaxrs/JwtAssertionInterceptor.java

```diff
@@ -139,30 +140,21 @@ public void filter(final ContainerRequestContext requestContext) throws IOExcept
 
             if (!claims.getAudience().contains(audience.toASCIIString())) {
                 LOG.warn("Audience {} did not match {} for {}", claims.getAudience(), audience, requestContext.getUriInfo());
-                requestContext.abortWith(Response.status(Status.UNAUTHORIZED)
-                    .header(HttpHeaders.WWW_AUTHENTICATE, "JWT")
-                    .entity("audience vertification failed")
-                    .build());
+                requestContext.abortWith(OAuthTokenResponse.unauthorized("invalid_audience", "Audience validation failed", "Bearer").getResponse());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Bearer" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)